server/e2e/api/specs/asset.e2e-spec.ts

```diff
@@ -1,1151 +0,0 @@
-  });
-
-  describe('PUT /asset/stack/parent', () => {
-    beforeEach(async () => {
-      const { status } = await request(server)
-        .put('/asset')
-        .set('Authorization', `Bearer ${user1.accessToken}`)
-        .send({ stackParentId: asset1.id, ids: [asset2.id, asset3.id] });
-
-      expect(status).toBe(204);
-    });
-
-    it('should require authentication', async () => {
-      const { status, body } = await request(server).put('/asset/stack/parent');
-
-      expect(status).toBe(401);
-      expect(body).toEqual(errorStub.unauthorized);
-    });
-
-    it('should require a valid id', async () => {
-      const { status, body } = await request(server)
-        .put('/asset/stack/parent')
-        .set('Authorization', `Bearer ${user1.accessToken}`)
-        .send({ oldParentId: uuidStub.invalid, newParentId: uuidStub.invalid });
-
-      expect(status).toBe(400);
-      expect(body).toEqual(errorStub.badRequest());
-    });
-
-    it('should require access', async () => {
-      const { status, body } = await request(server)
-        .put('/asset/stack/parent')
-        .set('Authorization', `Bearer ${user1.accessToken}`)
-        .send({ oldParentId: asset4.id, newParentId: asset1.id });
-
-      expect(status).toBe(400);
-      expect(body).toEqual(errorStub.noPermission);
-    });
-
-    it('should make old parent child of new parent', async () => {
-      const { status } = await request(server)
-        .put('/asset/stack/parent')
-        .set('Authorization', `Bearer ${user1.accessToken}`)
-        .send({ oldParentId: asset1.id, newParentId: asset2.id });
-
-      expect(status).toBe(200);
-
-      const asset = await api.assetApi.get(server, user1.accessToken, asset2.id);
-      expect(asset.stack).not.toBeUndefined();
-      expect(asset.stack).toEqual(expect.arrayContaining([expect.objectContaining({ id: asset1.id })]));
-    });
-
-    it('should make all childrens of old parent, a child of new parent', async () => {
-      const { status } = await request(server)
-        .put('/asset/stack/parent')
-        .set('Authorization', `Bearer ${user1.accessToken}`)
-        .send({ oldParentId: asset1.id, newParentId: asset2.id });
-
-      expect(status).toBe(200);
-
-      const asset = await api.assetApi.get(server, user1.accessToken, asset2.id);
-      expect(asset.stack).not.toBeUndefined();
-      expect(asset.stack).toEqual(expect.arrayContaining([expect.objectContaining({ id: asset3.id })]));
-    });
-  });
-
-  const getAssetIdsWithoutFaces = async () => {
-    const assetPagination = usePagination(10, (pagination) =>
-      assetRepository.getWithout(pagination, WithoutProperty.FACES),
-    );
-    let assets: AssetEntity[] = [];
-    for await (const assetsPage of assetPagination) {
-      assets = [...assets, ...assetsPage];
-    }
-    return assets.map((a) => a.id);
-  };
-
-  describe(AssetRepository.name, () => {
-    describe('getWithout', () => {
-      describe('WithoutProperty.FACES', () => {
```

Comment:
FYI, I have removed this set of tests. There is no way to test them in e2e and I'm not sure I can about adding them back anywhere else.